docs/adr/0014-use-eclipse-orion-as-editor.md

```diff
@@ -0,0 +1,31 @@
+# Use Eclipse Orion as editor
```

Comment:
Consistent spelling of headings.

In this file, headings are written in title case, so please write `Editor` in this heading.